<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTabWidget" name="tabWidget">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QFrame" name="frame">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QTextEdit" name="textEdit_config"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1199</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Views">
    <property name="title">
     <string>&amp;Views</string>
    </property>
    <addaction name="actionProperties"/>
    <addaction name="actionOutputs"/>
   </widget>
   <widget class="QMenu" name="menuSo_lve">
    <property name="title">
     <string>So&amp;lve</string>
    </property>
    <addaction name="action_Load_Gmsh_mesh_file"/>
    <addaction name="actionRun"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Views"/>
   <addaction name="menuSo_lve"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_Properties">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="0">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCommandLinkButton" name="commandLinkButton">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolBox" name="toolBox">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="page_Mesh">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>453</width>
          <height>390</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Mesh/Boundary conditions</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_mesh"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Mesh file</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QToolButton" name="toolButton_Load_mesh">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Number of BC</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QSpinBox" name="spinBox_nbBC"/>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QTableWidget" name="tableWidget_BC"/>
         </item>
         <item row="5" column="0" colspan="4">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_goSolver">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="go-next">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Solver">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>453</width>
          <height>390</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Solver</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="8" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton_goInit">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="go-next"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spinBox_cpu">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Time</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="5" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Step</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2" rowspan="3">
             <widget class="QLineEdit" name="lineEdit_timeEnd">
              <property name="text">
               <string>0.05</string>
              </property>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>s</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>saving rate</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" rowspan="2">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="lineEdit_timeStep">
              <property name="text">
               <string>0.00001</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="2">
             <widget class="QLineEdit" name="lineEdit_timeStart">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3" rowspan="2">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>s</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>End</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3" rowspan="2">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>s</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>s</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLineEdit" name="lineEdit_timeRate">
              <property name="text">
               <string>0.001</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_timeInteg">
           <item>
            <property name="text">
             <string>&lt;Time integration method&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Euler (1st order forward)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Runge-Kutta (4th order)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QComboBox" name="comboBox_elemType">
           <item>
            <property name="text">
             <string>&lt;Element type&gt;</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lagrange</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Isoparametric</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Number of thread</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Initialization">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>453</width>
          <height>390</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Initialization</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Number of initial values</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_nbInit"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTableWidget" name="tableWidget_Init">
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>60</number>
           </attribute>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="toolButton_goObs">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="go-next">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Observers">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>453</width>
          <height>390</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Observers</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Number of observers</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_nbObs"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QTableWidget" name="tableWidget_Obs">
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>60</number>
           </attribute>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="toolButton_goSrc">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset theme="go-next">
               <normaloff>.</normaloff>.</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Sources">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>453</width>
          <height>390</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Sources</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="1" column="0" colspan="2">
          <widget class="QTableWidget" name="tableWidget_Src">
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>60</number>
           </attribute>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Number of sources</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBox_nbSrc"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_Output">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Output</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_Outlet">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_add_term">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>New terminal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_rm_term">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <bold>true</bold>
           <underline>false</underline>
          </font>
         </property>
         <property name="text">
          <string>Remove terminal</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTermWidget" name="termwidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Quit"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset theme="window-close"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="preferences-other"/>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionOutputs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="go-down"/>
   </property>
   <property name="text">
    <string>Outputs</string>
   </property>
  </action>
  <action name="actionSave_case">
   <property name="icon">
    <iconset theme="document-save-as"/>
   </property>
   <property name="text">
    <string>Save case</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset theme="media-playback-start"/>
   </property>
   <property name="text">
    <string>&amp;Run</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Load_Gmsh_mesh_file">
   <property name="icon">
    <iconset theme="folder-open"/>
   </property>
   <property name="text">
    <string>&amp;Load Gmsh mesh file</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTermWidget</class>
   <extends>QWidget</extends>
   <header location="global">qtermwidget5/qtermwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionProperties</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidget_Properties</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOutputs</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockWidget_Output</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>721</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_Load_mesh</sender>
   <signal>clicked()</signal>
   <receiver>action_Load_Gmsh_mesh_file</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
